Competition/src/main/cpp/Arm.cpp

```diff
@@ -0,0 +1,98 @@
+#include "Arm.h"
+
+Arm::Arm() : armMtrLeft{ArmConstants::TALON_ID_LEFT_ARM}, 
+             armMtrRight{ArmConstants::TALON_ID_RIGHT_ARM} {
+
+}
+
+Arm::Arm(frc::XboxController* controller) : operator_controller(controller), 
+                                            armMtrLeft{ArmConstants::TALON_ID_LEFT_ARM}, 
+                                            armMtrRight{ArmConstants::TALON_ID_RIGHT_ARM} {
+    InitArm();
+}
+
+Arm& Arm::GetInstance() {
```

Comment:
You do not need this function. The super class does this for you. Can remove the definition in the header file, and delete this function from the implementation